boards/DIM/Src/Io/Io_SoftwareWatchdog.c

```diff
@@ -17,8 +17,5 @@ void Io_SoftwareWatchdog_TimeoutCallback(SoftwareWatchdogHandle_t watchdog)
 {
     BREAK_IF_DEBUGGER_CONNECTED();
 
-    const uint8_t watchdog_id = Io_SharedSoftwareWatchdog_GetTaskId(watchdog);
-    App_CanAlerts_SetAlert(DIM_ALERT_WATCHDOG_TIMEOUT, true);
-    App_CanTx_DIM_WatchdogTimeout_TaskName_Set((RtosTaskName)watchdog_id);
-    Io_CanTx_DIM_WatchdogTimeout_SendAperiodic();
+    App_CanTx_DIM_Warnings_WatchdogTimeout_Set(true);
```

Comment:
We should still have a signal that says which task caused the watchdog to timeout